nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -34,7 +34,6 @@
   - `services.mattermost.listenAddress` has been split into {option}`services.mattermost.host` and {option}`services.mattermost.port`. If your `listenAddress` contained a port, you will need to edit your configuration.
   - Mattermost now supports peer authentication on both MySQL and Postgres database backends. Updating {option}`system.stateVersion` to 25.05 or later will result in peer authentication being used by default if the Mattermost server would otherwise be connecting to localhost. This is the recommended configuration.
   - The Mattermost module will produce eval warnings if a database password would end up in the Nix store, and recommend alternatives such as peer authentication or using the environment file.
-  - Mattermost's entire test suite is now enabled by default, which will extend build time from sources by up to an hour. A `withoutTests` passthru has been added in case you want to skip it.
```

Comment:
> Mattermost's entire test suite is now enabled by default

Are we not running Mattermost's entire test suite by default?

Reply:
No, it's just in passthru.tests now, to avoid an hour-long build (since it depends on both mysql and postgres). It was good for stabilizing unstable, but any changes to it can be made separately.

Reply:
Ok, sounds good.